mmv1/products/bigqueryanalyticshub/Subscription.yaml

```diff
@@ -0,0 +1,150 @@
+# Copyright 2025 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'Subscription'
+description: A Bigquery Analytics Hub subscription
+references:
+  guides:
+    'Official Documentation': 'https://cloud.google.com/bigquery/docs/analytics-hub-introduction'
+  api: 'https://cloud.google.com/bigquery/docs/reference/analytics-hub/rest/v1/projects.locations.subscriptions'
+base_url: 'projects/{{project}}/locations/{{location}}/subscriptions'
+self_link: 'projects/{{project}}/locations/{{location}}/subscriptions/{{subscription_id}}'
+create_url: 'projects/{{project}}/locations/{{location}}/dataExchanges/{{data_exchange_id}}/listings/{{listing_id}}:subscribe'
+immutable: true
+import_format:
+  - 'projects/{{project}}/locations/{{location}}/subscriptions/{{subscription_id}}'
+custom_code:
+  post_create: templates/terraform/post_create/analytics_hub_subscription.go.tmpl
+  post_import: templates/terraform/post_import/analytics_hub_subscription.go.tmpl
+examples:
+  - name: 'bigquery_analyticshub_subscription_basic'
+    primary_resource_id: 'subscription'
+    primary_resource_name: 'fmt.Sprintf(\"tf_test_my_data_exchange%s\",
+      context[\"\
+      random_suffix\"]), fmt.Sprintf(\"tf_test_my_listing%s\",
+      context[\"random_suffix\"\
+      ])'
+    region_override: 'US'
+    vars:
+      data_exchange_id: 'my_data_exchange'
+      listing_id: 'my_listing'
+      destination_dataset_id: 'destination_dataset'
+      description: 'example data exchange'
+parameters:
+  - name: 'data_exchange_id'
+    type: String
+    description: |-
+      The ID of the data exchange. Must contain only Unicode letters, numbers (0-9), underscores (_). Should not use characters that require URL-escaping, or characters outside of ASCII, spaces.
+    url_param_only: true
+    required: true
+  - name: 'listing_id'
+    type: String
+    description: |-
+      The ID of the listing. Must contain only Unicode letters, numbers (0-9), underscores (_). Should not use characters that require URL-escaping, or characters outside of ASCII, spaces.
+    url_param_only: true
+    required: true
+  - name: 'location'
+    type: String
+    description: |
+      The name of the location this data exchange subscription.
+    url_param_only: true
+    required: true
+    custom_flatten: 'templates/terraform/custom_flatten/bigquery_dataset_location.go.tmpl'
+    diff_suppress_func: 'tpgresource.CaseDiffSuppress'
+properties:
```

Comment:
Looks like missing two output-only `linkedDatasetMap` and `linkedResources`

Reply:
added